README.md

```diff
@@ -1,135 +1,36 @@
-# Auction system
+# INTRO
 
-System architecture:
+This is a sample app of the Lagom Framework.
 
-# Item service
+## Running: Prerequisites
 
-Manages the description and auction status (created, auction, completed, cancelled) of an item.
+You will need to download and run an Elastisearch server:
```

Comment:
It meant doing too much sbt-ing to end up invoking `java.lang Process` and hoping the user would have `tar`, `gzip` installed and then discovering the OS... Plus potential changes on download locations of binary names in ES. The app starts without ES, and when ES is available, then you can also search.